l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -229,8 +234,8 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
             // slither-disable-next-line arbitrary-send-eth
             sharedBridge.bridgehubDepositBaseToken{value: msg.value}(
                 _request.chainId,
+                baseTokenAssetId[_request.chainId],
```

Comment:
Not critical to fix, but this is a bit of waste of gas:

- the address basetoken is already read at the top
- now we do another cold read of basically the same value, but slightly transformed. 

Not sure if calling NTV to get assetid from base token will be cheaper though...